package-lock.json

```diff
@@ -4486,9 +4486,9 @@
       }
     },
     "eslint-utils": {
-      "version": "1.4.2",
-      "resolved": "https://registry.npmjs.org/eslint-utils/-/eslint-utils-1.4.2.tgz",
-      "integrity": "sha512-eAZS2sEUMlIeCjBeubdj45dmBHQwPHWyBcT1VSYB7o9x9WRRqKxyUoiXlRjyAwzN7YEzHJlYg0NmzDRWx6GP4Q==",
+      "version": "1.4.0",
+      "resolved": "https://registry.npmjs.org/eslint-utils/-/eslint-utils-1.4.0.tgz",
+      "integrity": "sha512-7ehnzPaP5IIEh1r1tkjuIrxqhNkzUJa9z3R92tLJdZIVdWaczEhr3EbhGtsMrVxi1KeR8qA7Off6SWc5WNQqyQ==",
```

Comment:
@kevinlee12, Somehow I ended up on this and now I cannot undo this through commit (As pre_commit_hook will undo the change.)

Do you have any suggestion on how I can fix this?

Reply:
I used `--no-verify` flag.

Reply:
Typically, I’d use `git checkout HEAD — package-lock.json` to undo those changes. For this instance, it might have been that the git merge didn’t include this change.

Reply:
/cc @ankita240796

Reply:
>  For this instance, it might have been that the git merge didn’t include this change.

Yeah, this happened because:

- git pull upstream develop raised merge conflicts 
- Resolved merge conflict
- git commit to merge the changes but it also removed the changed in package-lock.json

> git checkout HEAD — package-lock.json

I think this will also have to go through the `git commit`, right?

Reply:
Yeah, I have a feeling it might be since that will have to go through the pre commit check (which would have undone this change), especially that you have a git conflict.